hphp/runtime/vm/jit/vasm-instr.h

```diff
@@ -1109,6 +1110,7 @@ struct shlq { Vreg64 s, d; VregSF sf; Vflags fl; }; // uses rcx
 /*
  * arm intrinsics.
  */
+struct andzi { Immed64 s0; Vreg64 s1, d; VregSF sf; Vflags fl; };
```

Comment:
In @cmuellner's original implementation of this, he used `andqi64{}`, which I thought was fairly descriptive, whereas `z` is not.  I would go with that.

(cf. https://github.com/facebook/hhvm/pull/7851)